boxsdk/object/base_object.py

```diff
@@ -169,63 +169,6 @@ def __ne__(self, other):
     def __hash__(self):
         return hash((self._object_id, self._item_type))
 
-    def _paging_wrapper(self, url, starting_index, limit, factory=None):
```

Comment:
I guess removing unused code will increase the test coverage (by percentage of lines covered), but this seems like it could be a separate commit